tests/tests.py

```diff
@@ -1,26 +1,44 @@
+import os, sys
 import logging
 
 import matplotlib
-matplotlib.use('Agg')
+if os.environ.get('DISPLAY','') == '':
+    logging.warning('No display found. Using non-interactive Agg backend')
+    matplotlib.use('Agg')
 
 import numpy as np
-from scipy.misc import imread
+from scipy.ndimage import imread
 from matplotlib import pyplot as plt
 
-import morphsnakes
+sys.path += [os.path.abspath('.'), os.path.abspath('..')]  # Add path to root
```

Comment:
Is this absolutely necessary? I find modifying the sys.path unsettling…

Reply:
well if you run just this test from `tests` folder and you did not install the packege before, it won't see the `morphsnakes` and consecutively it fails